src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -90,1016 +92,612 @@ private static IntPtr LoadNativeLibrary()
 
             private static void PlatformInitialize()
             {
-                LoadFunctionPointers();
+                NativeLibrary.SetDllImportResolver(Assembly.GetExecutingAssembly(), DllImportResolver);
+
+                s_gdipModule = LoadNativeLibrary();
```

Comment:
Is `s_gdipModule` still used anywhere? The only usage I see of it is here.

Reply:
I'll double check